package.json

```diff
@@ -86,11 +80,14 @@
         "@commitlint/cli": "^16.2.3",
         "@commitlint/config-conventional": "^16.2.1",
         "@nguniversal/builders": "^12.1.3",
-        "@nrwl/cli": "13.8.2",
-        "@nrwl/eslint-plugin-nx": "13.8.2",
-        "@nrwl/node": "13.8.2",
-        "@nrwl/tao": "13.8.2",
-        "@nrwl/workspace": "13.8.2",
+        "@nrwl/angular": "13.8.6",
+        "@nrwl/cli": "13.8.6",
+        "@nrwl/eslint-plugin-nx": "13.8.6",
+        "@nrwl/jest": "13.8.6",
+        "@nrwl/js": "13.8.6",
```

Comment:
[13.8.6-release](https://github.com/nrwl/nx/releases/tag/13.8.6) introduces the bug fix "fix(misc): support workspaces using a root tsconfig.json instead of tsconfig.base.json".